src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests.cs

```diff
@@ -7754,11 +7754,11 @@ static void M(object o)
             compilation.VerifyDiagnostics(); // we now recognize the extension method even without the assembly-level attribute
 
             var assembly = compilation.Assembly;
-            Assert.Equal(assembly.GetAttributes().Length, 0);
+            Assert.Equal(0, assembly.GetAttributes().Length);
             var type = compilation.GlobalNamespace.GetMember<NamedTypeSymbol>("E");
-            Assert.Equal(type.GetAttributes().Length, 0);
+            Assert.Equal(0, type.GetAttributes().Length);
             var method = compilation.GlobalNamespace.GetMember<NamedTypeSymbol>("E").GetMember<PEMethodSymbol>("M");
-            Assert.Equal(method.GetAttributes().Length, 0);
+            Assert.Equal(0, method.GetAttributes().Length);
```

Comment:
thanks so much.  it's a particular annoyance of mine when tests mix up 'expected' and 'actual'.

Reply:
What was really nice here is it was fairly easy. I just enabled the rule, ran fix all and it just worked.